File: src/problems/remove-duplicates-from-sorted-array/boilerplate-full/function.py
#USER_CODE_HERE
import sys
from typing import List

# Set recursion limit higher for recursive problems
sys.setrecursionlimit(2000)

if __name__ == "__main__":
    # Input reading for the specific problem
    nums = list(map(int, input().split())) # List of integers from space-separated input

    sol = Solution()
    
    # Call the solution function
    result = sol.removeDuplicates(nums)
    
    # Print the result
    print(result)

File: src/problems/remove-duplicates-from-sorted-array/boilerplate-full/function.java
//USER_CODE_HERE
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int numsSize = sc.nextInt();
        List<Integer> nums = new ArrayList<>();
        for (int i = 0; i < numsSize; i++) nums.add(sc.nextInt());
        Solution sol = new Solution();
        int result = sol.removeDuplicates(nums);
        System.out.println(result);
    }
}